.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Open Source Programs Run Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Run tests
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: osp
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
```

Comment:
Can't we use a matrix to test it with different versions

Reply:
I tried for stable version.
But any specific reasons for testing for multiple versions?

Reply:
Then we may need to add the versions supported in the documentation itself.